students/cbrown/Lesson 5/mailroom_part3.py

```diff
@@ -0,0 +1,117 @@
+#mailroom part 3: adding exceptions & comprehension
+#!/usr/bin/env python3
+#Mailroom Script Part 3. Upgrading with comprehensions & exceptions
+import sys
+import tempfile
```

Comment:
tempfile imported but not used

Reply:
Hi Charles,

Thank you very much for the feedback. I have modified the program and uploaded a newer version. I found the bug that caused the error. When a new donor name was being added to the dictionary (if it didn't already exist) I was adding just a float value to the dictionary instead of a list. So the next time the user went to add a second value to that new donor entry it was trying to append a float value to another float value instead of a list. 

New changes have fixed issue.

Thanks,

Chris